src/unstract/llmwhisperer/client.py

```diff
@@ -190,6 +191,7 @@ def whisper(
             ocr_provider (str, optional): The OCR provider. Can be "advanced" or "basic". Defaults to "advanced".
             line_splitter_tolerance (float, optional): The line splitter tolerance. Defaults to 0.4.
             horizontal_stretch_factor (float, optional): The horizontal stretch factor. Defaults to 1.0.
+            encoder (str): The character encoding to use for processing the text. Defaults to "ISO-8859-1".
```

Comment:
@ShoaibMajidDar Please rename variable to `encoding` as that is more accurate.
Also we can set default encoding to `utf-8` itself.